### DISCUSSION TITLE
Add 'Keep result' option when right clicking on a simulation result.

### DISCUSSION THREAD

**Discussion by ferrucci-franco** [OpenModelica/OpenModelica#10961] [Ideas]
Hi there,

I was wondering if you think it would be a good idea to add a 'Keep result' option when you right-click on a simulation result. 

The final goal would be to be able to change a model parameter directly on the simulation tab, re-simulate (using 'Re-simulate' option, without re-compiling the model), but keep the previous simulation with the old parameters. This would allow for a straightforward comparison of the same model with different parameters.

Currently, this is possible in a manual fashion, by:
a. Going into the simulation temp folder and copy the .mat file elsewhere (or copy+rename in the same folder).
b. Loading the simulation results: File-> Open Result File(s).

Thanks!

**Comment by adeas31** [comment]
What you want is already possible. You just need to generate a new result file.

Click on `Re-simulate Setup` and then go to `Output` tab and set a new name for the result file e.g., `A.mat`. This will keep the original result and load the new result file as well.
